src/components/SVGIcon.js

```diff
@@ -37,6 +37,15 @@ const getPath = (name, props) => {
           <path {...props} d="m91 180h90v-90h-90zm30-60h30v30h-30zm0 0" />
         </>
       );
+    case "github":

```

Comment:
"github" move to constants

Reply:
I'm planning on introducing constants once there are a high number of icons.